sdk/rms_sdk/Platform/Http/DnsServerResolverQt.cpp

```diff
@@ -108,9 +108,7 @@ std::string DnsServerResolverQt::lookup(const std::string& dnsRequest) {
       a.exec();
       return result;
   }
-  else{
```

Comment:
Wasn't this a bug that Satin fixed?

Reply:
Yes. You don't need the else clause here. Go through the entire changed file to see the logic.